src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
```

Comment:
Enum constant names should start with a capital letter. Please update the `Shape` enum to use capital letters for the enum keys.